### ISSUE TITLE
Forza Horizon 5 crashes and glitches

### ISSUE THREAD

**Issue by AlexBelyaninov** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#92]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Forza Horizon 5

### Game Platform [Required]

- [X] Steam
- [ ] Origin
- [X] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Bethesda.net
- [ ] Other (please specify)

### Processor / Processor Number [Required]

1135g7

### GPU Driver Version [Required]

- [X] 30.0.100.9955 (*latest*)
- [ ] 30.0.100.9894
- [ ] 30.0.100.9864
- [ ] 30.0.100.9862
- [ ] 30.0.100.9805
- [ ] 30.0.100.9684
- [ ] 30.0.100.9667
- [ ] 27.20.100.9616
- [ ] 27.20.100.9466
- [ ] 27.20.100.9316
- [ ] 27.20.100.9313
- [ ] 27.20.100.9168
- [ ] 27.20.100.9128

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [ ] Windows 11
- [ ] Windows 10 21H2 (*Insider Preview*)
- [X] Windows 10 21H1 (*May 2021 Update*)
- [ ] Windows 10 20H2 (*October 2020 Update*)
- [ ] Windows 10 20H1 (*May 2020 Update*)
- [ ] Windows 10 19H2 (*November 2019 Update*)
- [ ] Windows 10 19H1 (*May 2019 Update*)
- [ ] Other (Please specify)

### Description and steps to reproduce [Required]

Store version issues:
0 Game crashes 10-30 minutes after you start it. Screen freezes, VRAM usage grows rapidly, hits 100% and game closes. Happens every time. To replicate just run the game and stand in one place. Within 30 minutes you'll get a crash
https://photos.app.goo.gl/YbgD5Fw3FGkuMTxQ9

1 regardless of settings some sprites have black outline like letters on license plate
https://photos.app.goo.gl/1r6Nfkcs2UeZJPiu8

2 when car standing still, blinking artifacts appear on screen regardless of settings

3 enabling Lens effect (off->medium) causes whole screen to flash colors when light source (sun) is on screen https://www.youtube.com/watch?v=EajmGr4FPrw

4 enabling Deformable Terrain Quality (off->medium) causes input delay, frametimes increase and performance loss disproportional to what it suppose to be

Steam version issues:

Only 1, 3 and 4 out of the previous list.

So basically steam version is a bit glitchy but playable. Store version unplayable

### Device / Platform

Onexplayer

### Game Settings

tried all settings, only specified in description matter

**Comment by mva94** [comment]
@AlexBelyaninov do you have some pictures to speed up the process?

**Comment by AlexBelyaninov** [comment]
> @AlexBelyaninov do you have some pictures to speed up the process?

issues 1 and 2 https://photos.app.goo.gl/1r6Nfkcs2UeZJPiu8
issue 3 https://www.youtube.com/watch?v=EajmGr4FPrw

**Comment by thealgorithmc64** [comment]
On the latest driver i am having the following issues (tested on three tigerlake devices including the win3)

If settings other than very low are used,  eventually the screen will start to strobe purple/green colors and then colored tiling overlays cover most of the display.

**Comment by AlexBelyaninov** [comment]
> On the latest driver i am having the following issues (tested on three tigerlake devices including the win3)
> 
> If settings other than very low are used, eventually the screen will start to strobe purple/green colors and then colored tiling overlays cover most of the display.

This is Lens Effects. Keep them disabled and you can go up the settings

**Comment by evang-x** [comment]
Hello @AlexBelyaninov can you please test latest driver version 30.0.101.1069 to see if crash still occurs?

I'm also having trouble reproducing the lens effect issue. I've seen it appear before but it has been sporadic for me. Can I get a SS of the graphic and video settings you are using? Thank you.
